site/templates/application/footer/nav.jet

```diff
@@ -9,5 +10,7 @@
         {{ end }}
       </ul>
     {{ end }}
+    
+    {{ yield footerSocialMediaButtons() }}
```

Comment:
IMO, this should be outside the nav in [this file](https://github.com/shift72/core-template/blob/develop/site/templates/application/footer/footer.jet).

Reply:
Still think this should be in the footer file.